src/components/Option.js

```diff
@@ -74,7 +74,7 @@ const Option = ({ option }) => {
   let position = location.state.position || 0;
 
   const handleClick = e => {
-    if (e.type === 'click' || e.code === 'Enter') {
```

Comment:
Good catch here! Definitely remember coding this and `e.code` working, but 🤷. Thanks for catching and fixing!

Reply:
🤷🤷🤷🤷🤷